comp/process/agent/agentimpl/agent_linux_test.go

```diff
@@ -149,6 +150,9 @@ func TestProcessAgentComponentOnLinux(t *testing.T) {
 				}))
 			}
 
+			// reset agent module global variable "Once" to ensure Enabled() function runs for each unit test
+			agent.Once = sync.Once{}
```

Comment:
[minor] Put this in the defer section at the start of the test as this is really test cleanup.